File: 6/6.php
<?php
$payasos =  readline("Introduce numero de payasos: ");
$munecas = readline("Introduce numero de muñecas: ");
define('PESO_PAYASO',0.112);
define('PESO_MUNECA',0.075);

if(is_numeric($payasos) && is_numeric($munecas)){
    
    $total = (intval($payasos) * PESO_PAYASO) + (intval($munecas) * PESO_MUNECA);
    $total = floatval($total);
    echo "El peso total del paquete enviado sera en Kg: ".round($total,2);
}else{
    echo "Los valores deben ser numéricos";
}

File: 6/6.py
try:
    payasos =  int(input("Introduce numero de payasos: "))
    munecas = float(input("Introduce numero de muñecas: "))
    PESO_PAYASO = 0.112
    PESO_MUNECA = 0.075
    if(type(payasos)==int and type(munecas)==float):
            total = (payasos * PESO_PAYASO) + (munecas * PESO_MUNECA)
            print("El peso total del paquete enviado sera en Kg: " + str(round(total,2)))
except ValueError:
    print("Los valores deben ser numéricos")